backend/userauths/migrations/0001_initial.py

```diff
@@ -28,7 +28,7 @@ class Migration(migrations.Migration):
                 ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
                 ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
                 ('date_joined', models.DateTimeField(default=django.utils.timezone.now, verbose_name='date joined')),
-                ('username', models.CharField(max_length=100, unique=True)),
+                ('username', models.CharField(max_length=50, unique=True)),
```

Comment:
Any particular reason for changing the **max_lenght**?

Reply:
Also make suer that you ran proper migration before marge to avoid **unique_key_constraint** error

1. python manage.py migrate your_app zero
2. python manage.py makemigrations your_app
3. python manage.py migrate your_app